src/layout/Input/config.ts

```diff
@@ -11,6 +12,20 @@ export const Config = new CG.component({
     renderInAccordionGroup: false,
   },
 })
+  .addTextResource(
+    new CG.trb({
+      name: 'prefix',
+      title: 'Prefix',
+      description: 'Prefix shown next to the input field',
+    }),
+  )
+  .addTextResource(
+    new CG.trb({
+      name: 'suffix',
+      title: 'Suffix',
+      description: 'Suffix shown next to the input field',
+    }),
+  )
```

Comment:
Maybe we should add that these are different to number formatting? Or, perhaps people will find out soon enough.

Reply:
And 'next to' -> 'before'/'after'? Nice for those that don't immediately know what prefix/suffix means.